src/browser-installer/firefox/utils.ts

```diff
@@ -0,0 +1,13 @@
+const firefoxChannels = ["stable", "nightly"];
+
+export const normalizeFirefoxVersion = (version: string): string => {
+    return version.includes(".") ? version : `${version}.0`;
+};
```

Comment:
`@puppeteer/browsers` can't handle browser versions in format "115", so we add ".0" suffix if it isn't exist. That way it works